tests/scenarios/transform-complex-txn.spec.ts

```diff
@@ -273,15 +273,23 @@ describe('Complex transaction with many nested inner transactions', () => {
     expect(algodTxns.blockMetadata).toMatchInlineSnapshot(`
       [
         {
+          "currentProtocol": "https://github.com/algorandfoundation/specs/tree/abd3d4823c6f77349fc04c3af7b1e99fe4df699f",
```

Comment:
This will fail over time, we probably need to do a regex to replace the hash with a placeholder?